pkg/web/static/js/constants.js

```diff
@@ -275,4 +275,30 @@ export const networks = {
   "XZC": "ZCoin"
 };
 
-export const networksArray = Object.entries(networks).map(([value, text]) => ({ text, value }));
\ No newline at end of file
+export const networksArray = Object.entries(networks).map(([value, text]) => ({ text, value }));
+
+export const IDENTIFIER_TYPE = {
+  ADDRESS_TYPE_CODE_MISC: 'Unspecified',
+  ADDRESS_TYPE_CODE_HOME: 'Residential',
+  ADDRESS_TYPE_CODE_BIZZ: 'Business',
+  ADDRESS_TYPE_CODE_GEOG: 'Geographic',
+  LEGAL_PERSON_NAME_TYPE_CODE_LEGL: 'Legal',
+  LEGAL_PERSON_NAME_TYPE_CODE_SHRT: 'Short',
+  LEGAL_PERSON_NAME_TYPE_CODE_TRAD: 'Trading',
+  LEGAL_PERSON_NAME_TYPE_CODE_MISC: 'Unspecified',
+  NATIONAL_IDENTIFIER_TYPE_CODE_MISC: 'Unspecified',
+  NATIONAL_IDENTIFIER_TYPE_CODE_ARNU: 'ARNU',
+  NATIONAL_IDENTIFIER_TYPE_CODE_CCPT: 'Passport',
+  NATIONAL_IDENTIFIER_TYPE_CODE_RAID: 'RAID',
```

Comment:
This refers to registration authority identifier; but I think RAID is fine since it will be understandable to the folks who are using it (it is extremely uncommon).